$OpenBSD: patch-lib_efi_loader_efi_boottime_c,v 1.5 2019/07/28 06:57:20 jsg Exp $

f69d63fae281ba98c3d063097cf4e95d17f3754d in 2019.04 and later stopped
flushing/disabling caches on armv7 which is required to load
OpenBSD/armv7 kernels on at least i.MX 6.

Index: lib/efi_loader/efi_boottime.c
--- lib/efi_loader/efi_boottime.c.orig
+++ lib/efi_loader/efi_boottime.c
@@ -2875,8 +2875,6 @@ efi_status_t EFIAPI efi_start_image(efi_handle_t image
 	if (ret != EFI_SUCCESS)
 		return EFI_EXIT(EFI_INVALID_PARAMETER);
 
-	efi_is_direct_boot = false;
-
 	image_obj->exit_data_size = exit_data_size;
 	image_obj->exit_data = exit_data;
 
